game/models.py

```diff
@@ -1,3 +1,32 @@
+from django.contrib.auth.models import User
 from django.db import models
+from game.igdb_wrapper import IGDBRequestsHandler
 
-# Create your models here.
+
+class Game(models.Model):
+    """Game model"""
+
+    game_id = models.IntegerField()
+
+    def get_game_info_musts(self):
```

Comment:
nitpick: better to name this function `get_game_info_by_id`

Reply:
Done👌